Python/bytecodes.c

```diff
@@ -2642,14 +2631,15 @@ dummy_func(
                     PyObject *next = PyList_GET_ITEM(seq, it->it_index++);
                     PUSH(Py_NewRef(next));
                     JUMPBY(INLINE_CACHE_ENTRIES_FOR_ITER);
-                    DISPATCH();
+                    goto end_for_iter_list;  // End of this instruction
                 }
                 it->it_seq = NULL;
                 Py_DECREF(seq);
             }
             STACK_SHRINK(1);
             Py_DECREF(it);
             JUMPBY(INLINE_CACHE_ENTRIES_FOR_ITER + oparg + 1);
+         end_for_iter_list:
```

Comment:
```suggestion
        end_for_iter_list:
```

Reply:
Honestly I don't know what our style rule is for how far to indent labels.

Reply:
I always dedent them one level. I don't know if I've seen -3 spaces before, so it just felt like a typo to me.